cli-output/src/cli_output.rs

```diff
@@ -1532,55 +1532,28 @@ impl fmt::Display for CliProgramAuthority {
 #[serde(rename_all = "camelCase")]
 pub struct CliUpgradeableProgram {
     pub program_id: String,
-    pub program_executable: bool,
-    pub program_lamports: u64,
     pub programdata_address: String,
-    pub programdata_lamports: u64,
-    pub programdata_authority: String,
-    pub programdata_slot: u64,
-    pub programdata_data_len: usize,
-    pub programdata_program_len: usize,
-    pub use_lamports_unit: bool,
+    pub authority: String,
+    pub last_upgrade_slot: u64,
+    pub program_len: usize,
 }
 impl QuietDisplay for CliUpgradeableProgram {}
 impl VerboseDisplay for CliUpgradeableProgram {}
 impl fmt::Display for CliUpgradeableProgram {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         writeln!(f)?;
-        writeln_name_value(f, "Program Address:", &self.program_id)?;
-        writeln_name_value(f, "  Executable:", &self.program_executable.to_string())?;
-        writeln_name_value(
-            f,
-            "  Balance:",
-            &build_balance_message(self.program_lamports, self.use_lamports_unit, true),
-        )?;
-        writeln_name_value(f, "  ProgramData Address:", &self.programdata_address)?;
-        writeln_name_value(f, "    Authority:", &self.programdata_authority)?;
-        writeln_name_value(
-            f,
-            "    Balance:",
-            &build_balance_message(self.programdata_lamports, self.use_lamports_unit, true),
-        )?;
-        writeln_name_value(
-            f,
-            "    Total Length:",
-            &format!(
-                "{:?} ({:#x?}) bytes",
-                self.programdata_data_len, self.programdata_data_len
-            ),
-        )?;
+        writeln_name_value(f, "Program Id:", &self.program_id)?;
+        writeln_name_value(f, "ProgramData Address:", &self.programdata_address)?;
```

Comment:
```suggestion
        writeln_name_value(f, "Program Data Address:", &self.programdata_address)?;
```

Reply:
This is referred to as ProgramData everywhere so I'm going to keep it.